### DISCUSSION TITLE
NEW EXPLOIT!!! -- SKIOVOX

### DISCUSSION THREAD

**Discussion by Su4o** [3kh0/ext-remover#799] [Polls JOIN TITANIUM NETWORK TO SHARE PLEASE]
Yeah, here we go. (NOTE: This is not my exploit, it is akabutnice. Written by bypassi) 

here's the link to the tutorial. Sadly didn't work for me, but should work for others. [SKIOVOX](https://www.skiovox.com/skiovox.pdf)

**Comment by Ender-Ghost-Craft** [comment]
I was literally just going to ask them to add this

**Comment by 3kh0** [comment]
Adding this right now

**Comment by S-PScripts** [comment]
> Yeah, here we go. (NOTE: This is not my exploit, it is akabutnice. Written by bypassi)
> 
> here's the link to the tutorial. Sadly didn't work for me, but should work for others. [SKIOVOX](https://www.skiovox.com/skiovox.pdf)

me with no kiosk apps :(

**Reply by 3kh0** [reply]
:(

**Comment by KandyKaneMarvel** [comment]
Does anybody have suggestions for VPNs to use with this? Hola kinda sucks.

**Reply by 3kh0** [reply]
just look up any free VPN app on the chrome web store

**Reply by leahanderson1** [reply]
try sweetvpn, that was the one i found that worked for me

**Reply by Hilololsusy** [reply]
Just use any lmao, or just go on a proxy

**Reply by malapro2** [reply]
you got a proxy that works? my district has blocked just about everything, including  coding with chrome. if you've got a way to sign into app inventor without google, that could work, but i haven't found one.

**Reply by S-PScripts** [reply]
check all 1,000 of these https://github.com/S-PScripts/chromebook-utilities/blob/main/Prxes/Pr*xi*s%20Links.MD

hopefully at least 1 works...

**Reply by malapro2** [reply]
none of them work, my school is hella strict

**Reply by BlueJayNinja0909** [reply]
blud ust use a phone hotspot it fixed my problem with schol bolock even with skiovox

**Reply by 3kh0** [reply]
@S-PScripts sorry but I have to

💀 roblox

**Reply by S-PScripts** [reply]
https://raw.githack.com/3kh0/3kh0-assets/main/roblox/index.html
amogus funni ip address mem

**Reply by Legend7832** [reply]
Just use turbo vpn extension

**Comment by Shootoutgetaway** [comment]
does anyone know why every google site is blocked and how to fix (classroom, youtube, webstore, etc)

**Reply by BlueJayNinja0909** [reply]
School tires fixing this exploit

**Reply by BlueJayNinja0909** [reply]
Use hotspot go to chrome:network then network select then render and use different WiFi then school

**Comment by synapt0** [comment]
Ik this is an old forum but currently on ChromeOS 119 and it seems 2B and 2C have stopped working for me after the update. (I’m unable to use 2A.)

**Reply by leahanderson1** [reply]
(skiovox is patched)

**Reply by synapt0** [reply]
Damn. I didn’t know since I didn’t use my chromebook for shit over winter break lol.

**Reply by yeetmaster210** [reply]
i found a way to make it work and my chromebook is on version 120

**Reply by leahanderson1** [reply]
> i found a way to make it work and my chromebook is on version 120

kid named https://github.com/bypassiwastaken/skiovox-helper/issues/106

**Reply by yeetmaster210** [reply]
no for me my school found a way to block the chrome spoken feedback thing, so the way i do it is i just spam the open in settings button and after a while the screen will show up for like half a second but if you click the tab bar it over rides the kiosk app tab and thats how i do it

**Reply by synapt0** [reply]
How exactly do you do it? I’m lost.

**Reply by Aka-but-nice** [reply]
Just for this exact reason, I developed a method in case this happened. On the sign in screen, click "Add new account" with Wi-Fi off, it will show a network error screen. Then, open the kiosk app of your choice (any chrome app), turn on Chrome vox when the network error shows. Click Control + Alt + T click resources, then click any link there. Click the link, "Sign in as existing user", then sign in.

**Comment by Aka-but-nice** [comment]
:scream:
